docs/Manual/Deployment/Kubernetes/Drain.md

```diff
@@ -407,7 +421,7 @@ much data is stored in the pod, your mileage may vary, moving a terabyte
 of data can take considerably longer!
 
 If the optional step of
-[cleaning out a DBserver manually](#clean-out-a-dbserver-manually-optional)
+[cleaning out a DBserver manually](#clean-out-a-dbserver-manually)
 has been performed beforehand, the grace period can easily be reduced to 60
 seconds - at least from the perspective of ArangoDB, since the server is already
 cleaned out, so it can be dropped readily and there is still no risk.
```

Comment:
@neunhoef It still says "optional" here. Not sure if the entire sentence should be rewritten...

Reply:
It should, hang on...

Reply:
Done.